dashboard/lib/policies/active_record_roles.rb

```diff
@@ -16,4 +16,12 @@ def self.get_reading_role_name
     configurations = ActiveRecord::Base.configurations.configs_for(env_name: Rails.env, include_replicas: true)
     configurations.find(&:replica?)&.spec_name&.to_sym || get_writing_role_name
   end
+
+  # Get the name of the ActiveRecord "role" used for connecting to the reporting
+  # database which is a clone of our primary database meant for large / long
+  # queries usually used for data analysis. This role should not be used when
+  # handling web requests.
+  def self.get_reporting_role_name
+    :primary_reporting
```

Comment:
Just nitpicking naming here, but perhaps `primary_reporting` is more appropriate? The three connections are:

* primary
* primary_replica
* (reporting)

honestly, `primary_replica_reporting` might even be best, as it's a specific configuration applied to the same target as `primary_replica`.

I suspect that `reporting` is unanimously the second best name choice, and that we'll debate for weeks on what the first best choice is, so this isn't a blocking comment.

Reply:
I think the Rails multi-database convention is `my_database_replica` to denote that a database is a read replica of the one named `my_database`, so I think `my_database_reporting` follows that convention.